Control / Display

ikpeice opened the pull request:
Display Screen control added
Score increment corrected 
Added eye to dino image

ikpeice committed:
Changes to be committed:
	modified:   .gitignore
	modified:   dinoterm/game.py
	modified:   dinoterm/main.py
	modified:   dinoterm/play.py

ikpeice committed:
added getScreen_dimension() to fetch the height
and with of the console screen

	modified:   dinoterm/game.py
	modified:   dinoterm/main.py
	modified:   dinoterm/play.py

ikpeice committed:
On branch master
Your branch is up to date with 'origin/master'.
Changes to be committed:
	modified:   dinoterm/game.py
	modified:   dinoterm/main.py
	modified:   dinoterm/play.py

busayojee commented:
I am currently reviewing for package purpose. Would merge and publish as soon as packaging is done. Thanks Emmanuel😁

busayojee committed:
Update main.py

Just touched it up a bit.... It works fine overall, would do the same for game and play

busayojee committed:
Update .gitignore

busayojee committed:
Update game.py

busayojee commented on dinoterm/game.py:
I just changed the height of the obstacles to be a bit taller

busayojee commented on .gitignore:
I misspelled pycache so it wasn't adding it to gitignore

busayojee commented on dinoterm/main.py:
discarded relative imports which i had used for packaging in the original code. would revert to it in a bit.
got rid of variable game_width.
called getScreen_dimension in the main function, because for some odd reasons i was getting some errors.
So review this changes and let me know if you're okay with them so I can merge them

ikpeice replied:
ok

busayojee committed:
Update play.py

all changes can be found in the review

busayojee commented on dinoterm/play.py:
Just very little changes were made here by me

busayojee reviewed (commented):
Just let me know if you're okay so i can package and release version 1.0.1

ikpeice reviewed (commented):
##Yes, you can merge. Its looking good
